doc/source/train/getting-started-pytorch.rst

```diff
@@ -80,6 +80,7 @@ Compare a PyTorch training script with and without Ray Train.
     .. group-tab:: PyTorch + Ray Train
 
         .. code-block:: python
+            :emphasize-lines: 12, 18, 27, 42, 44-49
```

Comment:
Can we make this consistently include or exclude the comments?

For example:

This one is currently not highlighted:
```
# [1] Prepare model.
```

This one is:
```
# [4] Configure scaling and resource requirements.
```